chatbot/serializers.py

```
@@ -0,0 +1,364 @@
+        پارامترها:
+            obj (ChatbotSession): نمونه‌ی جلسه‌ای که شمارش مکالمات مرتبط با آن انجام می‌شود.
+        
+        برگردانده:
+            int: تعداد مدل‌های Conversation مرتبط با جلسه (استفاده از رابطه معکوس `conversations`).
+        """
+        return obj.conversations.count()
+
+
+class MessageSerializer(serializers.ModelSerializer):
+    """
+    سریالایزر پیام‌ها
```

Ruff (0.12.2)

Docstring contains ambiguous `ه` (ARABIC LETTER HEH). Did you mean `o` (LATIN SMALL LETTER O)?

(RUF002)